src/algorithmForNN/fileUtils.py

```diff
@@ -10,11 +10,9 @@ def write_ones_to_file_by_index_list(index_list: list, path_to_new_label_file: s
     """
     Function changes those lines in the file, the point with the index of which belongs to any plane
     """
-    file = open(path_to_new_label_file, 'r')
-    list_strings = file.readlines()
-    file.close()
+    with open(path_to_new_label_file, 'r') as file:
+        list_strings = file.readlines()
     for index in index_list:
```

Comment:
it is better to use list_strings only in there block, so inside "with" block (add tabs)